.travis.yml

```diff
@@ -52,8 +52,9 @@ script:
   - |
       if [[ $TESTS == 'blocks-examples' ]]; then
         cd $TRAVIS_BUILD_DIR
-        git clone https://github.com/mila-udem/blocks-examples.git
+        git clone https://github.com/laurent-dinh/blocks-examples.git
```

Comment:
¯\_(ツ)_/¯
`blocks-example` is a separate repository yet it is tested by Travis on this repository, meaning that any interface change will result in an automatic fail unless I do this kind of modification.
I made this modification on a separate commit so that we can revert when the time comes.